homeassistant/components/bsblan/__init__.py

```diff
@@ -29,6 +29,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         entry.data[CONF_HOST],
         passkey=entry.data[CONF_PASSKEY],
         port=entry.data[CONF_PORT],
+        username=entry.data[CONF_USERNAME],
+        password=entry.data[CONF_PASSWORD],
```

Comment:
For existing users this will miss in ConfigEntry. You should probably use `get` here

Reply:
But you would need to redo the setup anyway? If setup is done you won't request these entries?

Reply:
Yes, I mean for users who already configured it without username and password. They won't redo setup